Description: Remove dependency on time crate.
 The dependency on time 0.1 is only needed for compatibility, since time 0.1
 is not currently in Debian sid, that shouldn't be a problem for us anymore.
 See: https://github.com/chronotope/chrono/commit/5115c6f5f93d112d48b618edfa1b47d004c39d10
Author: Peter Michael Green <plugwash@debian.org>

Index: chrono/Cargo.toml
===================================================================
--- chrono.orig/Cargo.toml
+++ chrono/Cargo.toml
@@ -85,9 +85,6 @@ version = "1.0.99"
 optional = true
 default-features = false
 
-[dependencies.time]
-version = "0.1.43"
-optional = true
 
 [dev-dependencies.bincode]
 version = "1.3.0"
@@ -115,11 +112,9 @@ clock = [
 default = [
     "clock",
     "std",
-    "oldtime",
     "wasmbind",
 ]
 libc = []
-oldtime = ["time"]
 std = []
 # unstable-locales = [
 #     "pure-rust-locales",
